src/logic/wallets/getWeb3.ts

```diff
@@ -128,6 +128,7 @@ export const reverseENSLookup = async (address: string): Promise<string> => {
   let name = ''
   let verifiedAddress = ''
 
+  // There is no resolver contract for some networks
```

Comment:
Btw, we have a feature flag for this – `FEATURES.DOMAIN_LOOKUP`.
Not sure if it makes sense to use for this, it's fine if it just throws. Just FYI.

Reply:
It looks like Avalanche has that feature [enabled](https://safe-client.gnosis.io/v1/chains/43114) even though fetching the resolver contract throws an error `ENS is not supported on network private`

Reply:
@katspaugh, can you remove it from the prod Django?

Reply:
I've double-checked on Avalanche, it doesn't work indeed. Removed the feature. ✅